packages/browser/src/index.ts

```diff
@@ -103,6 +108,40 @@ export default class LogtoClient {
     sessionStorage.setItem(this.logtoStorageKey, jsonItem);
   }
 
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  private get refreshToken() {
+    return this._refreshToken;
+  }
+
+  private set refreshToken(refreshToken: Nullable<string>) {
+    const refreshTokenKey = buildRefreshTokenKey(this.logtoStorageKey);
+
+    if (refreshToken) {
+      localStorage.setItem(refreshTokenKey, refreshToken);
+      this._refreshToken = refreshToken;
```

Comment:
return early early instead of using else

Reply:
Improved more than that. PTAL
```ts
  private set refreshToken(refreshToken: Nullable<string>) {
    this._refreshToken = refreshToken;

    const refreshTokenKey = buildRefreshTokenKey(this.logtoStorageKey);

    if (refreshToken) {
      localStorage.setItem(refreshTokenKey, refreshToken);

      return;
    }

    localStorage.removeItem(refreshTokenKey);
  }
```